core/server/api/subscribers.js

```diff
@@ -201,6 +224,7 @@ subscribers = {
 
         // Push all of our tasks into a `tasks` array in the correct order
         tasks = [
+            getSubscriberByEmail,
```

Comment:
This should happen after validation/permissions check.

Reply:
Moved it and added the `email` option to the validate call `apiUtils.validate(docName, {opts: ['id', 'email']})`